multiload/load-graph.c

```diff
@@ -106,27 +109,27 @@ load_graph_draw (LoadGraph *g)
     }
     //printf("max = %d ", maxnet);
     guint level = 0;
-    if (maxnet > g->net_threshold3) {
-      g->net_threshold = g->net_threshold3;
+    if (maxnet > multiload->net_threshold3) {
+      multiload->net_threshold = multiload->net_threshold3;
       level = 3;
     }
     else
-      if (maxnet > g->net_threshold2) {
-        g->net_threshold = g->net_threshold2;
+      if (maxnet > multiload->net_threshold2) {
+        multiload->net_threshold = multiload->net_threshold2;
         level = 2;
       }
       else {
-        g->net_threshold = g->net_threshold1;
+        multiload->net_threshold = multiload->net_threshold1;
         level = 1;
-        if (maxnet < g->net_threshold1)
+        if (maxnet < multiload->net_threshold1)
           level = 0;
       }
 
     //printf("level %d maxnet = %d ", level, maxnet);
-    maxnet = maxnet/g->net_threshold;
+    maxnet = maxnet/multiload->net_threshold;
```

Comment:
Beautiful warning :)
https://travis-ci.org/github/mate-desktop/mate-applets/jobs/708166154#L4353
```
load-graph.c:129:20: warning: Division by zero

    maxnet = maxnet/multiload->net_threshold;
```

Reply:
The cppcheck report may be wrong, so I don't think it needs to check whether` multiload->net_threshold > 1`
```
maxnet = maxnet / MAX (g->net_threshold, 1);
```